blockauth/views/basic_auth_views.py

```
@@ -93,7 +93,7 @@ def post(self, request):
         except Exception as e:
             blockauth_logger.error("User signup failed", sanitize_log_context(request.data, {"error": str(e)}))
             logger.error(f"Request failed: {e}", exc_info=True)
```

Ruff (0.15.9)

[warning] Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)